packages/date-picker/src/panel/time-range.vue

```diff
@@ -121,13 +131,14 @@
         format: 'HH:mm:ss',
         visible: false,
         selectionRange: [0, 2],
-        arrowControl: false
+        arrowControl: false,
+        selectableRange: []
       };
     },
 
     watch: {
       value(value) {
-        if (Array.isArray(value)) {
+        if (this.isValidRange(value)) {
```

Comment:
Do not check isValidRange.
If developer want to show an invalid range, component should show it.